app/components/autocomplete_selector/autocomplete_selector.js

```diff
@@ -198,10 +206,21 @@ export default class AutocompleteSelector extends PureComponent {
                         >
                             {text}
                         </Text>
+                        {!disabled && onClear && selected && (
+                            <CompassIcon
+                                name='close-circle'
+                                color={changeOpacity(theme.centerChannelColor, 0.5)}
+                                style={style.clearx}
+                                size={20}
+                                onPress={this.handleClear}
+                            />
+                        )}
```

Comment:
@mickmister have you tried wrapping this with a [TapGestureHandler](https://docs.swmansion.com/react-native-gesture-handler/docs/api/gesture-handlers/tap-gh) or another Touchable?

Reply:
@enahum I have tried that with `TapGestureHandler` and `TouchableWithoutFeedback`, but `this.handleClear` is never called when I press on the button. There's no reaction to pressing the button. I've tried the following:

```js
<TouchableWithoutFeedback
    onPress={this.handleClear}
>
    <CompassIcon
        name='close-circle'
        color={changeOpacity(theme.centerChannelColor, 0.5)}
        style={style.clearx}
        size={20}
    />
</TouchableWithoutFeedback>


<TouchableWithoutFeedback
    onPress={this.handleClear}
>
    <View style={style.clearx}>
        <CompassIcon
            name='close-circle'
            color={changeOpacity(theme.centerChannelColor, 0.5)}
            size={20}
        />
    </View>
</TouchableWithoutFeedback>

<TapGestureHandler
    onHandlerStateChange={this.handleClear}
>
    <CompassIcon
        name='close-circle'
        color={changeOpacity(theme.centerChannelColor, 0.5)}
        style={style.clearx}
        size={20}
    />
</TapGestureHandler>


<TapGestureHandler
    onHandlerStateChange={this.handleClear}
>
    <View style={style.clearx}>
        <CompassIcon
            name='close-circle'
            color={changeOpacity(theme.centerChannelColor, 0.5)}
            size={20}
        />
    </View>
</TapGestureHandler>
```